js/ui/html_editor/ui.html_editor.js

```diff
@@ -611,10 +621,22 @@ const HtmlEditor = Editor.inherit({
         return this._formDialog.popupOption.apply(this._formDialog, arguments);
     },
 
+    update: function() {
+        this._applyQuillMethod('update');
+    },
+
     focus: function() {
         this.callBase();
 
         this._applyQuillMethod('focus');
+    },
+
+    blur: function() {
+        const activeElement = domAdapter.getActiveElement();
+        if(this.$element().is(activeElement) || this.$element().find(activeElement).length) {
+            resetActiveElement();
+        }
```

Comment:
Let's unify it? Extract the same method from texteditor -> editor. It is worth separating the condition into a separate function to make it easier to overload